src/main/kotlin/org/axonframework/intellij/ide/plugin/api/MessageHandler.kt

```diff
@@ -0,0 +1,10 @@
+package org.axonframework.intellij.ide.plugin.api
+
+import com.intellij.psi.PsiMethod
+
+data class MessageHandler(
+        val element: PsiMethod,
+        val handlerType: MessageHandlerType,
+        val payloadType: String,
+        val processingGroup: String?,
```

Comment:
Why does every message handler have a `processingGroup`?

Reply:
I've been thinking about making the `MessageHandler` an interface and then have specific types such as `EventMessageHandler` that can carry additional information for that type. I will put it on the list :)

Reply:
Awesome, great thinking, @Morlack.
Otherwise, if we would require some form of default name for the class containing the message handlers, I'd opt for something like `componentName`.

Reply:
There is now an interface for the different type of handlers, each can define their own text/icon rendering based on input. Now only EventProcessingHandlers have a processingGroup